src/PythonBindings/VectorPyList.hpp

```diff
@@ -11,15 +11,30 @@
 
 namespace bp = boost::python;
 
+/*!
+ * \ingroup PythonBindingsGroup
+ *\brief A namespace containing functions for binding Python wrappers
+ *to their respective c++ classes and functions, as well as helper functions for
+ *these wrappers.
+ */
 namespace py_bindings {
+
+/*!
+ * \ingroup PythonBindingsGroup
+ * \brief Convert a bp::list of bp::object& to a std::vector
+ */
 template <typename T>
 std::vector<T> py_list_to_std_vector(const bp::object& iterable) {
   return std::vector<T>(bp::stl_input_iterator<T>(iterable),
                         bp::stl_input_iterator<T>());
 }
 
+/*!
+ * \ingroup PythonBindingsGroup
+ * \brief Convert a std::vector into a bp::list of bp::objects
+ */
 template <class T>
-bp::list std_vector_to_py_list(const std::vector<T>& vector) {
+bp::list std_vector_to_py_list(std::vector<T> vector) {
```

Comment:
why can you no longer take this as a `const&`? I would expect that to work

Reply:
Seems to work fine, I guess that's  probably something I changed in the last pull request and choose the wrong one when I fixed the merge conflicts, sorry.

Reply:
Nothing to apologize for! :)